microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -580,23 +603,44 @@ See also:
 * [chrome.runtime](https://developer.chrome.com/extensions/runtime) - retrieves the background page.
 
 Status:
-*  This checkbox is present in Microsoft Edge Canary 116.
-*  This checkbox is present in Microsoft Edge Stable 114.
+*  This checkbox is present in Microsoft Edge Canary 117.
+*  This checkbox is present in Microsoft Edge Stable 115.
+
+
+<!-- ====================================================================== -->
```

Comment:
@zoherghadyali @captainbrosset - need real sentence for new Experiment checkbox "Show warning about Self-XSS when pasting code"

Reply:
Self-XSS is a security attack vector that relies on bad actors tricking unsuspecting users into attacking themselves by running code.
For example, a bad actor could create a website that asks users to copy some JavaScript code into their clipboard, then open the Console tool, paste that code into the Console, and press Enter.
This can allow the bad actor to run code on the user's device with the user's privileges.

This new experiment checkbox seems to work in this way: when it's enabled, and the user pastes code into the Console tool, then a warning message appears in the Console. Looking at the source code, it seems like the warning is the following one:

> Warning: Do not paste code you do not understand or have not checked yourself into the DevTools console. This could allow attackers to steal your identity or take control of your computer. Please type 'allow pasting' below to allow pasting.

I have not yet seen this feature work however. I've tried on Chrome Canary with the experiment ON, but I'm not seeing a warning yet. Not sure what's wrong.

I hope this helps document the experiment.

Reply:
I'll enter fresh comment(s)